### ISSUE TITLE
FMUv1 hangs on boot

### ISSUE THREAD

**Issue by julianoes** [PX4/Firmware#5855]
FMUv1 fails booting with master. This was reported in http://discuss.px4.io/t/mpu-sensor-problem/1667.

`git bisect` points to c24046ca5bcfc220b579ab5229af8fbf5010816c
```
c24046ca5bcfc220b579ab5229af8fbf5010816c is the first bad commit
commit c24046ca5bcfc220b579ab5229af8fbf5010816c
Author: Lorenz Meier <lorenz@px4.io>
Date:   Sun Sep 4 20:55:50 2016 +0200

    FMUv1: Save resources in config
```

**Comment by LorenzMeier** [comment]
I reviewed the diff and compared to other configs - we came up with final numbers close to the tested original. I've undone the commit, fixed in 0ca397ab2046c2a2a71b3ca758e1cbb6f80a87fb.

**Comment by julianoes** [comment]
I wouldn't go as far as undoing the whole commit especially because it negatively affects tap.

I would go with just #5856.
